src/app.ts

```diff
@@ -427,9 +434,52 @@ function optimizeRow(
   }
 
   let genDescription = origDescription;
+  let genDescriptionScore = -1;
+  let genDescriptionEvaluation = 'No Evaluation';
+  let genDescriptionApproval = true;
+
   if (getConfigSheetValue(CONFIG.userSettings.feed.generateDescriptions)) {
     try {
+      genDescriptionApproval = false;
       genDescription = fetchDescriptionGenerationData(dataObj, genTitle);
+      if (getConfigSheetValue(CONFIG.userSettings.feed.evaluateDescriptions)) {
+        let retries = 0;
+        try {
+          retries = parseInt(
+            getConfigSheetValue(
+              CONFIG.userSettings.descriptionValidation.maxRetries
+            )
+          );
+        } catch (e) {
```

Comment:
Unnecessary - max retries should be an app-level setting, not a user setting, and should be fixed to 3 (it should already exist in config.ts if I'm not mistaken). I would even argue that we don't need a "maxRetryCount" at all, since the way FeedGen is built is that you could always "rerun" wherever you're not happy. So users could theoretically filter on "descriptionScore<4" and "regenerate". Consider removing it altogether.

If there is a reason behind keeping it a "user-level" setting (e.g. explicit customer request), then we should also use it as a "multiplier" for the cost estimation in the sheet (i.e. promptLength*retries = total number of chars to use for the cost calculation).

Reply:
As discussed, removing retries from logic and config.